CONTRIBUTING.md

```diff
@@ -12,6 +11,57 @@ This monorepository contains the different subprojects for it:
                     i.e. as _lambda functions_
 
 
+## Terminology
+
+An Exocom application consists of a number of collaborating services.
+For example, the _todo-application_ consists of a _user_ service for storing user accounts,
+a _todo_ service for storing todo items,
+an _html_ service for exposing the application's functionality via an HTML UI,
+a _rest_ service for exposing it via a REST API,
+and a _chat_ service for exposing it via a chatbot.
+
+"User" and "todo" are the __role__ that the respective services play in the application.
+A role can be played by different __service types__.
+For example, you could use the _FacebookUserService_ service type
+to play the role of the _user_ service for your application in production,
+and you could use a _MockUserService_ to play the same role for local testing.
+You can also use the same service type for different roles,
+for example use a generic _MongoService_ as a quick-and-dirty prototype
```

Comment:
The wording on this line seems weird, maybe something like this could work? 
`for example, *you could* use a generic...`